src/main/java/ai/elimu/web/content/word/WordCreateController.java

```diff
@@ -125,7 +126,7 @@ public String handleSubmit(
         if (existingWord != null) {
             result.rejectValue("text", "NonUnique");
         }
-        if(word.getText().contains(" ")) {
+        if (org.springframework.util.StringUtils.containsWhitespace(word.getText())){
```

Comment:
Do you mean to use `org.apache.commons.lang.StringUtils`?

Reply:
Can use that also. I'll use that in my next update.